flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/FlowModelGeneratorService.java

```diff
@@ -264,6 +267,34 @@ public CompletableFuture<JsonObject> getSuggestedComponents(SuggestedComponentRe
         });
     }
 
+    @JsonRequest
+    public CompletableFuture<FlowModelGeneratorResponse> flowModelDifference(FlowModelDifferenceRequest request) {
```

Comment:
Are we producing a difference or a flow model? if it is, the name `flowModelDifference` is not appropriate right?